content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -627,13 +631,38 @@ read-modify-write and/or patch are the following:
 It is strongly recommended for controllers to always "force" conflicts, since they
 might not be able to resolve or act on these conflicts.
 
+### Transferring Ownership
+
+In addition to the concurrency controls provided by [conflict
+resolution](#conflicts), Server Side Apply provides ways to perform coordinated
+field ownership transfers:
+
+From a user to a controller: First the user signals to the controller that it
+should take ownership of a field, usually by setting some other field that the
+controller observes. The controller then claims ownership of a field by applying
+a configuration containing the field, with force set to true. Once the user
+observes that the controller owns the field, they can remove the field from
+their applied configuration to complete the transfer. This ensures the field is
+not accidently removed from the live object, or reset to a default value, at any
+point of time during the transfer. This can be used, for example, to safely transfer
+ownership of the `replicas` field from a user to a controller while enabling
+Horizontal Pod Autoscaling on an object. Controllers should use apply, not
+update, to claim ownership of fields since only apply is able to claim ownership
+of a field even if the desired value is the same as the live value.
```

Comment:
Describe what to do if the controller doesn't actually make a write to take ownership of the field, for example if it is already at the correct value. (Basically, the user instructions below, but just make up a unique manager name and have it only own the one field; when the controller eventually makes a write, the made up manager will no longer own anything and will be removed, so there's no clean up necessary.)

Reply:
Thanks. I've incorporated this. The user->controller section now looks like:

```
First the user signals to the controller that it should take ownership of a
field, usually by setting some other field that the controller observes. The
controller then claims ownership of the field by applying a configuration
containing the field. This can be done in one of the following ways:

- If the controller wants to immediately set the value to the field, it can
apply with force set to true.
- If the controller does not want to change the value to the field immediately,
the controller should genereate a unique, temporary, field manager name, and
claim shared ownership of the field by applying a configuration with the field
set to its live value. Later, when the controller does want to write the value
of the field, it can then apply with force set to true using the field manager
name it typically uses, which will automatically delete the temporary field
manager without the need for any clean up.

Once the user observes that the controller owns the field, they can remove the
field from their applied configuration to complete the transfer. This ensures
the field is not accidently removed from the live object, or reset to a default
value, at any point of time during the transfer.

This approach can be used, for example, to safely transfer ownership of the
`replicas` field from a user to a controller while enabling Horizontal Pod
Autoscaling on an object. Controllers should use apply, not update, to claim
ownership of fields since only apply is able to claim ownership of a field even
if the desired value is the same as the live value.
```